scripts/node.sh

```diff
@@ -82,6 +82,18 @@ parse_params() {
   docker_network='axelarate_default'
   node_moniker="$(hostname | tr '[:upper:]' '[:lower:]')"
 
+  # Set the appropriate chain_id
+  if [ "$network" == "mainnet" ]; then
```

Comment:
`network` is set during the `while` loop, so this won't set the chain id appropriately based on the network. We should keep the if check as before, but not override the directory if it was already set.